### PR TITLE
more info on how to contribute

### PR DESCRIPTION
**Pull request by nickdesaulniers**
review? @mykmelez
continuation of Pull Request #316

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Fork on github https://github.com/mozilla/r2d2b2g
+2. clone your fork
+`git clone git@github.com:your_github_username_here/r2d2b2g.git`
+3. cd into your cloned fork `cd r2d2b2g`
+4. set r2d2b2g/master as upstream `git remote add --track master upstream
+https://github.com/mozilla/r2d2b2g.git`
+5. initialize submodules `git submodule init && git submodule update`
+6. make `make`
+7. initialize addon sdk `cd addon-sdk && . bin/activate`
```
Comment (mykmelez) [inline CONTRIBUTING.md:11]: Just a few more nits!

addon sdk -> Add-on SDK

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Note: to quit the instance of Firefox started by `make run`, use Firefox's
+Quit/Exit menu item, as aborting the task on the command line with Control-C
+will crash Firefox, and you'll lose changes to the apps on the Dashboard.
+
+## Using Custom Profiles ##
+
+If you need to use a custom Firefox profile, you can skip step 9 and run via
+the following command
+
+`cfx run --templatedir template/ --profiledir /path/to/arbitrary/profile/`
```
Comment (mykmelez) [inline CONTRIBUTING.md:24]: Now that we have make run PROFILE, make this say:

make run PROFILE=/path/to/arbitrary/profile/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+# Fixing Issues #
+
+Pick up an issue from the
+[issue tracker](https://github.com/mozilla/r2d2b2g/issues?state=open) and
+checkout a new branch that describes the issue.
```
Comment (mykmelez) [inline CONTRIBUTING.md:30]: Nit: checkout -> check out

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ex. `git checkout -b fixThatBrokenThingYouKnow`
+
+squashed commits preferred, then issue a pull request.
\ No newline at end of file
```
Comment (mykmelez) [inline CONTRIBUTING.md:33]: Nit: add a newline to the end of the file.

**Comment by nickdesaulniers** [comment]
Will issue new pull request

**Pull request closed without merging**